[2025-02-12T21:54:40.453+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T21:54:40.464+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-01T03:29:00+00:00 [queued]>
[2025-02-12T21:54:40.471+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-01T03:29:00+00:00 [queued]>
[2025-02-12T21:54:40.471+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T21:54:40.484+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-01 03:29:00+00:00
[2025-02-12T21:54:40.490+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=979) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T21:54:40.490+0000] {standard_task_runner.py:72} INFO - Started process 986 to run task
[2025-02-12T21:54:40.490+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-01T03:29:00+00:00', '--job-id', '758', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpsxs8eahw']
[2025-02-12T21:54:40.491+0000] {standard_task_runner.py:105} INFO - Job 758: Subtask fetch_and_store_crypto_data
[2025-02-12T21:54:40.521+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-01T03:29:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T21:54:40.581+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T03:29:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T03:29:00+00:00'
[2025-02-12T21:54:40.582+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T21:54:40.672+0000] {logging_mixin.py:190} INFO - Données récupérées : {'BTC': {'USD': 97595.27}, 'ETH': {'USD': 2685.35}}
[2025-02-12T21:54:40.678+0000] {logging_mixin.py:190} INFO - Données insérées dans PostgreSQL.
[2025-02-12T21:54:40.681+0000] {logging_mixin.py:190} INFO - Erreur lors de l'upload dans MinIO : S3 operation failed; code: NoSuchBucket, message: The specified bucket does not exist, resource: /crypto-data, request_id: 1823949C6E42CB3C, host_id: dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8, bucket_name: crypto-data
[2025-02-12T21:54:40.681+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T21:54:40.688+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T21:54:40.688+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-01T03:29:00+00:00, execution_date=20231001T032900, start_date=20250212T215440, end_date=20250212T215440
[2025-02-12T21:54:40.744+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T21:54:40.754+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T21:54:40.757+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
